docs/usage/desktop/twmenu.md

```diff
@@ -0,0 +1,54 @@
+# TWiLightMenu++
```

Comment:
I'm thinking `docs/usage/handheld` might be better.

Also, this will need to get added to `mkdocs.yml`.

Reply:
My rationale was that it's an addon software that runs on top of the devices OS but since people probably associate it with the Handheld more than with it's design mechanics I totally agree :D

Reply:
Looks like this still needs an addition to `mkdocs.yml`. I'll look into adding tests for when a page is "hidden" (hasn't been added to the index, yet).

Reply:
resolved with https://github.com/emmercm/igir/commit/0e3712ee6cd4c62f841e88260ab5085cf2d7eb58 i hope. I merged your changed from main and npm run docs:serve didn't fire a warning concerning unused files. Sorry for missing that and resolving the conversation. I think I fixed it in the wrong branch by mistake.